### DISCUSSION TITLE
[Idea] Scrollable Sidebar Icons

### DISCUSSION THREAD

**Discussion by Nick2bad4u** [runelite/runelite#16368] [Ideas]
I'm using so many plugins that use a sidebar that it is starting to run off my screen preventing me from using them.

Is there any way we can make this "Scrollable"?

Example:

![image](https://user-images.githubusercontent.com/20943337/224520709-b19b7a3c-a779-4cd6-b934-68c9695e9fc3.png)

**Comment by Nightfirecat** [comment]
We have a project in the works to update the look and feel library used by the client which, when completed, will include an update to the sidebar to allow it to expand to a second column when space runs out. We've discussed allowing it to scroll and decided the UX for that is not good, so that's not the approach we'll be taking to solve that issue.

**Reply by Nick2bad4u** [reply]
> We have a project in the works to update the look and feel library used by the client which, when completed, will include an update to the sidebar to allow it to expand to a second column when space runs out. We've discussed allowing it to scroll and decided the UX for that is not good, so that's not the approach we'll be taking to solve that issue.

Beautiful!

**Comment by dominio50** [comment]
Not sure if this has been suggested for this, tried looking in the discord for it couldn't find much. What if there was a button/tab where the settings wrench, profiles, plugin hub are, that houses a list of the sidebar buttons? When it's clicked they open the content of the sidebar tab. Not exactly fluent is what is possible with the UI but just providing a possible solution.
